recipes/treemix/meta.yaml

```diff
@@ -1,39 +1,38 @@
 package:
   name: treemix
-  version: 1.12
+  version: 1.13
 
 build:
-  number: 1
-  skip: True # [osx]
-  string: boost{{CONDA_BOOST}}__gsl{{CONDA_GSL}}_{{PKG_BUILDNUM}}
-
+  number: 0
+  string: boost{{ CONDA_BOOST }}_gsl{{ CONDA_GSL }}_{{ PKG_BUILDNUM }}
 
 source:
-  fn: treemix-1.12.tar.gz
-  url: https://bitbucket.org/nygcresearch/treemix/downloads/treemix-1.12.tar.gz
-  md5: 183f605751a0fa445e249db840907104
+  fn: treemix-1.13.tar.gz
+  url: https://bitbucket.org/nygcresearch/treemix/downloads/treemix-1.13.tar.gz
+  md5: 2a744026d3fd567d76104dd15e6a2887
 
 requirements:
   build:
+  - cmake
   - gcc
-  - zlib {{CONDA_ZLIB}}*
-  - gsl {{CONDA_GSL}}*
-  - boost {{CONDA_BOOST}}*
+  - blas
+  - zlib {{ CONDA_ZLIB }}*
+  - gsl {{ CONDA_GSL }}*
+  - boost {{ CONDA_BOOST }}*
 
   run:
   - libgcc
-  - zlib {{CONDA_ZLIB}}*
-  - gsl {{CONDA_GSL}}*
+  - blas
+  - zlib {{ CONDA_ZLIB }}*
+  - gsl {{ CONDA_GSL }}*
+  - boost {{ CONDA_BOOST }}*
 
-about:
-  home: http://pritchardlab.stanford.edu/software.html
-  license: GPLv3
-  summary: TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a set of populations
 test:
   commands:
-    - treemix --help | grep 1.12
+    - treemix --help | grep 1.13
 
-extra:
```

Comment:
I would keep this `extra` section in the updated `meta.yaml` if it's still valid.